setup.py

```diff
@@ -37,6 +37,7 @@
         "pyyaml",
         "requests >= 2",
```

Comment:
shouldn't this be 2.4 ?

Reply:
I haven't merged master back into the 9.3 release branch yet.